enterprise/cypher/acceptance-spec-suite/src/test/resources/cypher/features/DurationBetweenAcceptance.feature

```diff
@@ -0,0 +1,383 @@
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+
+Feature: DurationBetweenAcceptance
+
+    # TODO the commented lines should work too
+
+  Scenario: Should compute duration between two temporals
+    Given an empty graph
+    When executing query:
+      """
+      UNWIND [duration.between(date("1984-10-11"), date("2015-06-24")),
+              duration.between(date("1984-10-11"), localdatetime("2016-07-21T21:45:22.142")),
+              duration.between(date("1984-10-11"), datetime("2015-07-21T21:40:32.142+0100")),
+              duration.between(date("1984-10-11"), localtime("16:30")),
+              duration.between(date("1984-10-11"), time("16:30+0100")),
+
+              duration.between(localtime("14:30"), date("2015-06-24")),
+              duration.between(localtime("14:30"), localdatetime("2016-07-21T21:45:22.142")),
+              duration.between(localtime("14:30"), datetime("2015-07-21T21:40:32.142+0100")),
+              duration.between(localtime("14:30"), localtime("16:30")),
+              duration.between(localtime("14:30"), time("16:30+0100")),
+
+              duration.between(time("14:30"), date("2015-06-24")),
+              //duration.between(time("14:30"), localdatetime("2016-07-21T21:45:22.142")),
+              duration.between(time("14:30"), datetime("2015-07-21T21:40:32.142+0100")),
+              duration.between(time("14:30"), localtime("16:30")),
+              duration.between(time("14:30"), time("16:30+0100")),
+
+              //duration.between(localdatetime("2015-07-21T21:40:32.142"), date("2015-06-24")),
+              duration.between(localdatetime("2015-07-21T21:40:32.142"), localdatetime("2016-07-21T21:45:22.142")),
+              duration.between(localdatetime("2015-07-21T21:40:32.142"), datetime("2015-07-21T21:40:32.142+0100")),
+              //duration.between(localdatetime("2015-07-21T21:40:32.142"), localtime("16:30")),
+              //duration.between(localdatetime("2015-07-21T21:40:32.142"), time("16:30+0100")),
+
+              //duration.between(datetime("2014-07-21T21:40:36.143+0200"), date("2015-06-24")),
+              duration.between(datetime("2014-07-21T21:40:36.143+0200"), localdatetime("2016-07-21T21:45:22.142")),
+              duration.between(datetime("2014-07-21T21:40:36.143+0200"), datetime("2015-07-21T21:40:32.142+0100"))
+              //duration.between(datetime("2014-07-21T21:40:36.143+0200"), localtime("16:30")),
+              //duration.between(datetime("2014-07-21T21:40:36.143+0200"), time("16:30+0100"))
+              ] as d
+      RETURN d
+      """
+    Then the result should be, in order:
+      | d |
```

Comment:
Does this mean that no duration support exists yet? We can run these functions with no errors, but also get no durations produced. Seems like silently failing.

Reply:
Nononononono. We were just to lazy to code the expected result. This will be handled in a different PR.